README.md

```diff
@@ -1,6 +1,207 @@
-# terraform-vcf-metal-deployment
+# terraform-equinix-metal-vcf
 
-This Terraform module deploys a VMware Cloud Foundation (VCF) environment on Equinix Metal.
+[![run-pre-commit-hooks](https://github.com/equinix-labs/terraform-equinix-metal-vcf/actions/workflows/pre-commit.yaml/badge.svg)](https://github.com/equinix-labs/terraform-equinix-metal-vcf/actions/workflows/pre-commit.yaml)
+[![generate-terraform-docs](https://github.com/equinix-labs/terraform-equinix-metal-vcf/actions/workflows/documentation.yaml/badge.svg)](https://github.com/equinix-labs/terraform-equinix-metal-vcf/actions/workflows/documentation.yaml)
+
+`terraform-equinix-metal-vcf` is a minimal Terraform module that utilizes [Terraform providers for Equinix](https://registry.terraform.io/namespaces/equinix) to provision digital infrastructure and demonstrate higher level integrations.
+
+<img src="assets/8-VCF_on_Metal_ft_Metal_VRF.png" width="608" height="552" alt="Target Metal Architecture featuring Metal VRF for Underlay routing">
```

Comment:
Would it be worth breaking this up into smaller PRs so we can improve the readme incrementally while figuring out the workflow bits?  These intro lines and diagram in particular would be a big improvement to what's there right now.